src/stacks_wallet_provider.ts

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
big fan of what these examples are looking like

Reply:
this will make for a really easy client SDK as well if we ever wanted to do that.

Reply:
Yeah, we will probably have to lead the way and get something like this adopted in `stacks-wallet-web`.